src/ai/helpers/telemetry_helper.cs

```diff
@@ -18,22 +21,26 @@ class TelemetryHelpers
 
         internal static ITelemetry Instantiate(TelemetryConfig config, IProgramData programData)
         {
+            ITelemetry? telemetry = null;
             try
             {
-                switch (config.Type)
+                switch (config.Type?.ToLowerInvariant())
                 {
                     case "none":
                     default:
-                        return NoOpTelemetry.Instance;
+                        break;
+
                     case "aria":
-                        return new AriaTelemetry(config.Aria.TenantId, programData);
+                        telemetry = CreateTelemetry("Azure.AI.CLI.Telemetry.Aria.AriaTelemetry", "Azure.AI.CLI.Telemetry.Aria", config.Aria.TenantId, programData?.TelemetryUserAgent);
+                        break;
                 }
+
+                return telemetry ?? NoOpTelemetry.Instance;
             }
             catch (Exception ex)
             {
 #if DEBUG
-                ErrorHelpers.WriteLineMessage(
-                    "WARNING:", "Failed to instantiate telemetry", ex.Message);
+                ErrorHelpers.WriteLineMessage("WARNING:", "Failed to instantiate telemetry", ex.Message);
```

Comment:
Is it expected that as a dev, without setting `AI_CLI_ENABLE_TELEMETRY=1`, that i'd see this error line printed? I enlisted in the branch, and built without it, and I did see this message:

![image](https://github.com/Azure/azure-ai-cli/assets/1608910/8ad31ab2-685b-4217-9978-43e328039abe)

Reply:
I'm totally fine with that... just asking to make sure I understand it correctly. 

And, thus, people outside Microsoft that build it themselves would also see this. Which is totally fine.

Reply:
Hmm.. fair point. I will make this error message conditional on the environment variable being set to avoid confusion